### ISSUE TITLE
[Docs] Meshery UI: Add contributing instructions for end-to-end testing using Cypress

### ISSUE THREAD

**Issue by leecalcote** [meshery/meshery#5211]
#### Current State
The [contributing page for Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui) doesn't currently mention Cypress and end-to-end tests.

#### Desired State
https://docs.meshery.io/project/contributing/contributing-ui should include specifics of considerations that contributors should make while contributing to Meshery UI.

The labeler commenter for PRs matching `component/ui` can also be updated to include a hyperlink to this section of the docs.

Context:
https://layer5io.slack.com/archives/C011VPREG2Z/p1647265714076679

---

#### [Contributor Guides](https://docs.meshery.io/project/contributing) and Resources
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

**Comment by leecalcote** [comment]
// @MarioArriaga92

**Comment by subhamgcon** [comment]
@leecalcote Is anyone working on this issue? If no, Can I work on it?

**Comment by leecalcote** [comment]
@subhamgcon, yes, have you worked with Cypress before?

**Comment by subhamgcon** [comment]
> @subhamgcon, yes, have you worked with Cypress before?

No, but I wanna learn! Can you help me with how to get started with it?

**Comment by MarioArriaga92** [comment]
@leecalcote this has been in my tray for some time so I'll get it done this week.

@subhamgcon are you part of our slack workspace? We have a #cypress channel where we can help get you set up and tackle learning básicos of cypress and contributing to Meshery end-to-end testing, here's [ Slack Workspace invite link](https://join.slack.com/t/meshery/shared_invite/zt-rgbyt4kj-6PUUyqY9SfQGRSt_oZV9yQ)

**Comment by MarioArriaga92** [comment]
Getting my local setup for meshery docs, working!:
<img width="1423" alt="CleanShot 2022-03-23 at 09 32 09@2x" src="https://user-images.githubusercontent.com/23443575/159736434-98b9f694-c2eb-450b-8b91-d123dc7b8fed.png">

**Comment by MarioArriaga92** [comment]
@subhamgcon you should definitely try out these new docs and post on our slack workspace any useful feedback - https://docs.meshery.io/project/contributing/contributing-cypress
